### ISSUE TITLE
FilesystemAdapterDecorator incompatibility in tests.

### ISSUE THREAD

**Issue by mortenscheel** [getsentry/sentry-laravel#876]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

3.8.2

### Steps to reproduce

Laravel's own `assertExists()` method allows passing an array of paths:
```php
Storage::disk('local)->assertExists(['path1', 'path2']);
```

### Expected result

Test should pass when both paths exist.

### Actual result

```
TypeError: Sentry\Laravel\Features\Storage\SentryFilesystemAdapter::withSentry(): Argument #3 ($description) must be of type ?string, array given, called in /var/www/html/vendor/sentry/sentry-laravel/src/Sentry/Laravel/Features/Storage/FilesystemAdapterDecorator.php on line 13
/var/www/html/vendor/sentry/sentry-laravel/src/Sentry/Laravel/Features/Storage/FilesystemDecorator.php:41
/var/www/html/vendor/sentry/sentry-laravel/src/Sentry/Laravel/Features/Storage/FilesystemAdapterDecorator.php:13
```

**Comment by mortenscheel** [comment]
Thanks for the super fast fix!

**Comment by cleptric** [comment]
Released in [v4.4.1](https://github.com/getsentry/sentry-laravel/releases/tag/4.4.1).
